packages/Python/lldbsuite/test/lang/swift/implementation_only_imports/main_executable/Makefile

```diff
@@ -0,0 +1,21 @@
+LEVEL = ../../../../make
+
+EXE=main
+SWIFT_SOURCES=main.swift
+SWIFTFLAGS_EXTRAS= -I$(shell pwd)
+LD_EXTRAS=-L$(shell pwd) -lSomeLibrary
+
+all: SomeLibrary.swiftmodule main
+
+include $(LEVEL)/Makefile.rules
+
+SomeLibrary.swiftmodule: SomeLibrary.swift
+	$(MAKE) MAKE_DSYM=NO CC=$(CC) SWIFTC=$(SWIFTC) \
+		SWIFTFLAGS_EXTRAS="$(MOD$*_FLAGS)" \
```

Comment:
How is `$(MOD$*_FLAGS)` expanded?

Reply:
Oops, that's left over from what I copied from. I should simplify this.